.travis.yml

```diff
@@ -80,9 +79,9 @@ jobs:
 
     - stage: test
       <<: *stage_test_go
-      <<: *language_go_1_11
-    - <<: *stage_test_go
       <<: *language_go_1_12
```

Comment:
Would this be a good moment to remove the 1_12 testing here, as proposed in #2069 ?

Reply:
done.